logic/metrics.go

```diff
@@ -58,6 +58,7 @@ func CollectServerMetrics(serverID string, networkNodes []models.Node) *models.M
 			newServerMetrics.Connectivity[currNodeID] = models.Metric{
 				Connected: false,
 				Latency:   999,
+				PercentUp: 0,
```

Comment:
Why is this in the MQ PR?

Reply:
also the zero value for the struct will be 0 anyways